benchmark/t-test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+// Class for doing statistic test following a Student's t-distribution,
+// with default configuration: mu = 0, alternative = two.sided
+function TTest(leftData, rightData) {
+  this.left = this.createData(leftData);
+  this.right = this.createData(rightData);
+  this.df = this.left.size + this.right.size - 2;
+  this.dfhalf = this.df / 2;
+  this.mean = this.left.mean - this.right.mean;
+  this.commonVariance = ((this.left.size - 1) * this.left.variance +
```

Comment:
Don't assume common variance, my investigations shows that this is very rare. Use the Welch t-test.